### ISSUE TITLE
Failing to connect to Azure SQL Edge database

### ISSUE THREAD

**Issue by stevenhurwitt** [microsoft/azuredatastudio#13055]
- Azure Data Studio Version: 1.2.3

This is a little outside the scope of ADS, but I figured I'd post in case anyone had a similar issue and resolved it.

Steps to Reproduce:

1. Follow tutorial to set up Azure SQL Edge database [here](https://dbafromthecold.com/2020/04/22/running-azure-sql-edge-on-a-raspberry-pi/).

2. Connect to local IP of iotedge SQL Server using Azure Active Directory authentication.

Error message:

`A connection was successfully established with the server, but then an error occurred during the pre-login handshake. (provider: HTTP Provider, error: 0 - )`.

Basically I'm wondering if is there a way to debug this further? The logs give the same error message.

**Edit:** After doing some more digging, Trust Server Certificate has to be set to True under Advanced settings. Had to authenticate with SQL Login and the username was "sa".
